### ISSUE TITLE
Won't Install / Converge on Centos 8

### ISSUE THREAD

**Issue by nhillsdonuk** [sous-chefs/mariadb#300]
## :ghost: Brief Description

Default Centos 8 minimal install using stock parameters fails to find candidate for MariaDB-Server.

## :pancakes: Cookbook version

Latest

## :woman_cook:  Chef-Infra Version

Latest

## :tophat: Platform details

Centos 8 Virtual Machine installed with Centos 8 minimal.

## Steps To Reproduce
In recipe:

mariadb_server_install 'package' do
  setup_repo false
  password 'banana'
  action [:install, :create]
end

It searches for MariaDB-Server if not Debian - however Centos 8 only matches mariadb-server and is case sensitive.

Line 224 in mariadb/libraries/helpers.rb needs the quick fix.

## :police_car: Expected behavior

Install mariadb-server package from default Centos repo.

**Comment by IcemistPixie** [comment]
Hi,
This is not due to the package name. This is an issue with CentOS 8 & DNF. The package name in the mariadb.org yum repo's is still MariaDB-server, the one that you are finding in lower case is the one from the CentOS 8 provided repos. The issue is that dnf is not finding the packages in the mariadb.org repos for some reason.

Having looked into this again I see that the mariadb install [instructions](https://downloads.mariadb.org/mariadb/repositories/#distro=CentOS&distro_release=centos8-amd64--centos8&mirror=guzel&version=10.4) have been updated since I looked in November, and it now suggests adding an extra option to the yum repo file, to fix this. I have a working fix for this in my clone of this repo on a self hosted gitlab instance. I'll submit a pull request with a fix for this later on today.
